ArcGISRuntimeSDKQt_CppSamples/Scenes/ViewPointCloudDataOffline/ViewPointCloudDataOffline.cpp

```diff
@@ -0,0 +1,101 @@
+// [WriteFile Name=ViewPointCloudDataOffline, Category=Scenes]
+// [Legal]
+// Copyright 2018 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#include "ViewPointCloudDataOffline.h"
+
+#include "ArcGISTiledElevationSource.h"
+#include "Scene.h"
+#include "SceneQuickView.h"
+#include "PointCloudLayer.h"
+#include <QDir>
+
+#ifdef Q_OS_IOS
```

Comment:
Include `#include <QtCore/qglobal.h>` before using Qt's platform macros. 

This should be done to make sure the Qt platform macros are defined. See [here](https://github.com/KDE/clazy/blob/master/docs/checks/README-qt-macros.md).